docs/5_2_hybrid_co-simulation_api.adoc

```diff
@@ -70,25 +68,25 @@ The Co-Simulation master signals to the FMU that it supports and has recognized
+In this mode the Co-Simulation master is supposed to catch all events through the <<fmi3NewDiscreteStates>> function.
 
 If the early-return request of the Co-Simulation master is ignored by the FMU, then <<fmi3DoStep>> returns with `earlyReturn = fmi3False`.
 The master can start a resynchronization of FMUs at an event time, if the `currentCommunicationPoint` has passed the event time, the master can roll-back the FMU and repeat the step with a suitable `communicationStepSize` (if the FMU supports the roll-back).
 
-In *Event Mode* and only after <<fmi3DoStep>> returns with `earlyReturn = fmi3True` a <<fmi3NewDiscreteStates>> call returns <<fmi3EventInfo>> structure as a return argument.
-Not all elements of <<fmi3EventInfo>> are relevant and only the following elements may be considered:
+In *Event Mode* and only after <<fmi3DoStep>> returned with `earlyReturn = fmi3True` the function <<fmi3NewDiscreteStates>> may be called.
+Only the following output arguments may be considered:
 
-- when `newDiscreteStatesNeeded = true` it means that the master should stay in *Event Mode* and another call to <<fmi3NewDiscreteStates>> is required.
+- When `newDiscreteStatesNeeded = true`, the master should stay in *Event Mode* and another call to <<fmi3NewDiscreteStates>> is required.
 
-- when `nextEventTimeDefined = true` it means that an event time is available and the value is given by the `nextEventTime` value.
+- When `nextEventTimeDefined = fmi3True`, an event time is available and the value is given by `nextEventTime`.
 This is the case when the model can report in advance the accurate time of the next predictable time event.
 
-- when `terminateSimulation = true` it means that the model wants to stop integration and the Co-Simulation master should call <<fmi3Terminate>>.
+- When `terminateSimulation = fmi3True`, the model requested to stop integration and the Co-Simulation master should call <<fmi3Terminate>>.
 
-In *Event Mode* it is allowed to `fmi3Get{VariableType}` variable values of the FMU after the call of <<fmi3NewDiscreteStates>> and it is allowed to `fmi3Set{VariableType}` variable values before calling <<fmi3NewDiscreteStates>>.
-The FMU leaves *Event Mode* when the master calls <<fmi3EnterStepMode>> for that FMU.
+In *Event Mode* it is allowed to call `fmi3Get{VariableType}` after <<fmi3NewDiscreteStates>> has been called and it is allowed to call `fmi3Set{VariableType}` before calling <<fmi3NewDiscreteStates>>.
+The FMU leaves *Event Mode* when the master calls <<fmi3EnterStepMode>>.
 
-It is not allowed to call <<fmi3EnterEventMode>> or <<fmi3EnterStepMode>> for an FMU with interface `fmi3CoSimulation` or `fmi3ScheduledCoSimulation`.
+It is not allowed to call <<fmi3EnterEventMode>> or <<fmi3EnterStepMode>> for an FMU with interface type Basic Co-Simulation and Scheduled Co-Simulation.
```

Comment:
what was the reason to switch from or to and?

Reply:
Because it applies to both interface types. Do you want it changed?

Reply:
The second reads better - but it does not link to the section... so far we overdid linking, are we stopping that? I will close this as this is a different issue altogether